style.css

```diff
@@ -0,0 +1,338 @@
+/* @import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap"); */
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background: #e5e5e5;
+  margin: 0;
+  padding: 0;
+  font-weight: 400;
+  font-size: 16px;
+  color: black;
+}
+
+header {
+  padding: 0 24px 0;
+  background-color: #fff;
+  width: 375px;
+}
+
+.main {
+  padding: 24px 24px 1px;
+  border-bottom-left-radius: 80px;
+  margin-bottom: 0;
+  background-image: url("/images/headerbg.png");
+  background-size: cover;
+  background-position: 100% 10%;
+  background-repeat: no-repeat;
+  height: 650px;
+  width: 100%;
+  background-color: #fff;
+  margin-top: -2px;
+}
+
+.toolbar-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 30px;
+  width: 100%;
+}
+
+.nav-container {
+  width: 18px;
+  height: 12px;
+  left: 3px;
+  top: 6px;
+  color: #6070ff;
+}
+
+.logo {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 400;
+  text-align: left;
+  line-height: 24px;
+  color: #344563;
+  width: 100%;
+}
+
+.p {
+  width: 100%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 0;
+}
```

Comment:
- [ ]  Good job so far, but there is something you need to work on, but you are almost there. I noticed that your paragraphs are bolded, and since we are imitating the Figma design, your paragraph `font-family` should be the same. :wink: 

Your Design :point_down: 
![peter2](https://user-images.githubusercontent.com/88894525/192891100-be8577cf-c5ad-4059-8a82-e867c12a2429.png)

Figma Design :point_down: 
![figmaparagraph](https://user-images.githubusercontent.com/88894525/192891952-dba7917d-57bd-4ad4-bd1e-98800611df4b.png)

Reply:
@Alfredbis29 Thanks for the review. Please, kindly check my css on paragraphs. I actually used what is on figma design. I really do not know what next to do about it

Reply:
@Alfredbis29 Hello!

Reply:
fixed